.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "granulate-utils"]
 	path = granulate-utils
-	url = git@github.com:Granulate/granulate-utils.git
+	url  = https://github.com/Granulate/granulate-utils.git
```

Comment:
Revert please. By the way if `git submodule update` with the SSH URL fails for you, configuring Git SSH access will solve it.

Reply:
Done